stumpwm.asd

```diff
@@ -59,4 +59,7 @@
                (:file "remap-keys")
                ;; keep this last so it always gets recompiled if
                ;; anything changes
-               (:file "version")))
+               (:file "version"))
+  :in-order-to ((test-op (load-op "stumpwm-tests")))
+  :perform (test-op (o c)
+                    (uiop/package:symbol-call "FIASCO" "RUN-TESTS" 'stumpwm-tests)))
```

Comment:
Instead of adding this `:perform` to the `stumpwm` system, I think it should be in the `stumpwm-tests` system. Then the option above it would be:

```lisp
:in-order-to ((test-op (test-op "stumpwm-tests")))
```

Reply:
Thanks, updated the PR